more_itertools/more.py

```diff
@@ -3650,12 +3648,12 @@ def _sample_unweighted(iterator, k, strict):
     reservoir = list(islice(iterator, k))
     if strict and len(reservoir) < k:
         raise ValueError('Sample larger than population')
-    W = 1.0
+    w = 1.0
```

Comment:
NO!  The `W` was chosen for a reason.  See the paper referenced in the comments.  This was another blind substitution make without any understanding of the code or the author's intent.